### DISCUSSION TITLE
remote XP12/Datarefs across the LAN

### DISCUSSION THREAD

**Discussion by Glasairman** [MobiFlight/MobiFlight-Connector#1726] [Ideas]
Is this in your forward planning?
I want to run Mobiflight on the same machine running Air Manager
Thanks from Munich

**Comment by DocMoebiuz** [comment]
yes. there is already a branch from last year. I will likely integrate it into the new MobiFlight UI, hopefully in time for AERO2024.

**Comment by Glasairman** [comment]
Vielen Dank für die schnelle Antwort
David

Gesendet von AOL auf Android 
 
  Am Sa., März 2, 2024 at 0:48 schrieb Sebastian ***@***.***>:   


yes. there is already a branch from last year. I will likely integrate it into the new MobiFlight UI, hopefully in time for AERO2024.
